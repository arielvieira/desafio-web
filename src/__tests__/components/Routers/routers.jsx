import React from 'react';
import { shallow } from 'enzyme';

import Routes from '../../../routers';

describe('Routes', () => {
  it('should render Routes correctly', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toMatchSnapshot();
  });
});
