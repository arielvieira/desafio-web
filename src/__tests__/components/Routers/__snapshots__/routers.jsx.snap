// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render Routes correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/pullrequests/:creator/:repository"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
